src/AppInstallerCommonCore/Downloader.cpp

```diff
@@ -22,6 +23,9 @@ namespace AppInstaller::Utility
         IProgressCallback& progress,
         bool computeHash)
     {
+        // For AICLI_LOG usages with string literals.
+        #pragma warning(disable:26449)

```

Comment:
Do you know which line is causing this?  I would just like to understand it better.

Also the typical pattern would be:
```
#pragma warning(push)
#pragma warning(disable:26449)
<code>
#pragma warning(pop)
```

Reply:
It's being caused by the usages on lines 61, 65, 77, 93, 125 (in Downloader.cpp). I can't tell what those lines have in common that 30 does not.

Reply:
Hmm, feels like an erroneous warning if it is on the `AICLI_LOG` lines.  There is a `string` -> `string_view` function call in the macro, but the docs page for 26449 says it shouldn't trigger on those.  Strange...